src/cwe_checker_lib/src/analysis/forward_interprocedural_fixpoint.rs

```diff
@@ -283,3 +284,149 @@ pub fn create_computation_with_alternate_worklist_order<'a, T: Context<'a>>(
         default_value.map(NodeValue::Value),
     )
 }
+
+/// Generate a new computation from the corresponding context and an optional default value for nodes.
+/// Uses the Callee before Caller worklist order when computing the fixpoint.
+pub fn create_computation_with_callee_before_caller_worklist_order<'a, T: Context<'a>>(
```

Comment:
`bottom_up_worklist_order` is in my opinion a better name than `callee_before_caller_worklist_order`, since bottom-up analysis is a common term for this.